dev/bots/analyze.dart

```diff
@@ -1308,6 +1308,7 @@ Future<void> verifyRepositoryLinks(String workingDirectory) async {
     'dart-lang/webdev',
     'eseidelGoogle/bezier_perf',
     'flutter/devtools', // TODO(guidezpl): remove when https://github.com/flutter/devtools/issues/7551 is closed
+    'flutter/flutter', // TODO(fujino): https://github.com/flutter/flutter/issues/121564
```

Comment:
```suggestion
```

I am removing this for now @christopherfujino since there are tests that verify this is not in the list. 😓